apps/aechannel/src/aesc_offchain_state.erl

```diff
@@ -132,41 +139,38 @@ check_reestablish_tx(SignedTx, State) ->
 
 -spec check_update_tx(aetx_sign:signed_tx(), state(), aec_trees:trees(),
                       aetx_env:env(), map()) -> ok | {error, atom()}.
-check_update_tx(SignedTx, State, OnChainTrees, OnChainEnv, Opts) ->
-    check_update_tx_(none, SignedTx, State, OnChainTrees, OnChainEnv, Opts).
-
-check_update_tx_(F, SignedTx, #state{signed_tx = OldSignedTx}=State, OnChainTrees,
+check_update_tx(SignedTx, #state{signed_tx = OldSignedTx}=State, OnChainTrees,
                  OnChainEnv, Opts) when OldSignedTx =/= ?NO_TX ->
     lager:debug("check_update_tx(State = ~p)", [State]),
     Tx = aetx_sign:tx(SignedTx),
     {Mod, TxI} = aetx:specialize_callback(Tx),
     lager:debug("Tx = ~p", [Tx]),
-    case Mod:round(TxI) - 1 of
-        0 when OldSignedTx == ?NO_TX ->
-            lager:debug("previous round = 0", []),
-            check_update_tx_(F, Mod, TxI, State, OnChainTrees, OnChainEnv, Opts);
-        PrevRound ->
+    case Mod of
+        aesc_close_mutual_tx -> ok; % no particular check for a close mutual
+        _ ->
+            PrevRound = Mod:round(TxI) - 1,
+            true = PrevRound > 0, % assert not channel_create
```

Comment:
I checked that all code paths leading here specify tx type, and none is channel create. So comment on assertion makes sense. Ok.